jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_lifelog/pr2_tweet.launch

```diff
@@ -3,6 +3,7 @@
 
   <include file="$(find jsk_robot_startup)/lifelog/tweet.launch">
     <arg name="robot_name" value="PR2"/>
+    <arg name="image_topics" default="/edgetpu_human_pose_estimator/output/image" />
```

Comment:
please use default kinect image for this PR

Reply:
Reverted that commit.
Now PR2 uses default topics: `/kinect_head/rgb/image_rect_color` and `/wide_stereo/left/image_rect_color`